services-directions/build.gradle

```diff
@@ -8,9 +8,9 @@ dependencies {
   compileOnly dependenciesList.supportAnnotation
 
   // AutoValue
-  compileOnly dependenciesList.autoValue
```

Comment:
Why is this not needed anymore?

Reply:
`dependenciesList.autovalue` is annotation processor, we don't need to connect it one more time as `compileOnly`. Docs: https://github.com/google/auto/blob/master/value/userguide/index.md#with-gradle.

Reply:
reverted for consistency. I don't remember why I removed this one 🙂 
I guess everything still works because the project gets auto value transitively